analysis_options.yaml

```diff
@@ -6,6 +6,8 @@ analyzer:
     unused_import: error
     unused_local_variable: error
     dead_code: error
+  enable-experiment:
+    - non-nullable
```

Comment:
Note: this is not how we anticipate turning on NNBD in the long term.  See https://github.com/dart-lang/sdk/issues/38347.